apps/server/src/modules/task/controller/task.controller.ts

```diff
@@ -133,16 +142,6 @@ export class TaskController {
 		return response;
 	}
 
-	@Get(':taskId')
-	async findTask(@Param() urlParams: TaskUrlParams, @CurrentUser() currentUser: ICurrentUser): Promise<TaskResponse> {
-		this.FeatureTaskCardEnabled();
```

Comment:
in case its just about leaving out the feature flag, deleting this line would be enough. if the `users` property in task entity can stay, than the rest of the changes is not necessary

Reply:
You're right, that the change in functionality is just done by deleting the feature flag. The rest of the changes are architecture and clean code related. Especially we have an arch decision that authorization should be done in the UC, see: https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/authorisation.html